exercises/webapp/src/components/HackerNews.spec.js

```diff
@@ -0,0 +1,68 @@
+import {
+  mount
+} from '@vue/test-utils';
+import HackerNews from './HackerNews.vue';
+import NewsItem from './NewsItem.vue';
+
+const createWrapper = (options) => {
+  const defaults = {
+    propsData: {
+      initialNews: [{
+        id: 0,
+        title: 'Message 1',
+        voteCount: 2
+      }, {
+        id: 1,
+        title: 'Message 2',
+        voteCount: 4
+      }],
+    }
+  };
+  return mount(HackerNews, { ...defaults, ...options });
+};
+
+describe('Sorting Order', () => {
+  let wrapper;
+  beforeEach(() => {
+    wrapper = createWrapper({});
+  });
+
+  it('sorting order', async () => {
+    expect(wrapper.findAll('h2>.news-message').wrappers.map(w => w.text())).toEqual(['Message 2', 'Message 1']);
+    await wrapper.find('#switch-order-button').trigger('click');
+    expect(wrapper.findAll('h2>.news-message').wrappers.map(w => w.text())).toEqual(['Message 1', 'Message 2']);
+    await wrapper.find('#switch-order-button').trigger('click');
+    expect(wrapper.findAll('h2>.news-message').wrappers.map(w => w.text())).toEqual(['Message 2', 'Message 1']);
```

Comment:
Why so over-specific with `h2`? When you do that, your test case "knows" about it and it will break, e.g. when you decide to refactor `h2` to `h3`.